### PR TITLE
Added installation instructions to README

### PR DESCRIPTION
**Pull request by doekman**
I struggled with this issue myself, so I decided to just put in the effort and add it myself. I hope this is helpful. If I need to make changes, no problem...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,18 @@ Github](http://github.com/belluzj/cosmic-sans-neue/issues) if you stumble upon
 bad design or rendering problems (with screen shot if possible), or if you need
 more characters, or if you want to compliment me (I love compliments).
 
+Installation
+------------
+
+You can [download the latest version][] and install it by hand. In the `NoLoopK` variant, the looped lowercase `k` is replaced with a straight version. The `LargeLineHeight` variant is especially useful for users of accented capitals. For more info, see the [CHANGELOG](CHANGELOG.md).
```
Comment (belluzj) [inline README.md]: You probably want to link to https://github.com/belluzj/fantasque-sans/releases/latest

**Comment by belluzj** [comment]
Thanks for this! If you can just put the link where you intended to put one, then I'll merge.

**Comment by belluzj** [comment]
Thanks again :)

**Comment by doekman** [comment]
No worries. For some reason, my latest commit didn't seem to merge (called `typo`, fixing _installion_ to _installation_)

**Comment by belluzj** [comment]
I fixed it by hand. I may have clicked on "merge" at the same time that you were adding your commit.

**Pull request merged**